decidim-core/app/cells/decidim/author_cell.rb

```diff
@@ -23,43 +26,57 @@ def author_name
       options[:author_name_text] || model.name
     end
 
-    def show
-      render
+    def display_name
+      model.deleted? ? t("decidim.profile.deleted") : decidim_sanitize(author_name)
     end
 
-    def profile
-      render
+    def flag_user
+      render unless current_user == model
     end
 
-    def profile_inline
-      render
+    def perform_caching?
+      true
     end
 
-    def date
-      render
+    def raw_model
+      model.try(:__getobj__) || model
     end
 
-    def flag_user_modal
-      render
-    end
+    def context_actions_options
+      return unless options.has_key?(:context_actions)
+      return [] if options[:context_actions].blank?
 
-    def flag
-      render
+      @context_actions_options ||= options[:context_actions].map(&:to_sym)
     end
 
-    def flag_user
-      render unless current_user == model
+    private
+
+    def layout
+      @layout ||= LAYOUTS.include?(options[:layout]) ? options[:layout] : :default
     end
 
-    def withdraw
-      render
+    def show_icons?
+      layout != :compact
     end
 
-    def perform_caching?
-      true
+    def context_actions
+      # REDESIGN_PENDING: deprecated?
+      # return [] unless actionable?
```

Comment:
@entantoencuanto I think this comment can be removed

Reply:
You are right, I've checked that the deprecation refers to the `actionable` not the `context_action` method